README.md

```diff
@@ -21,22 +20,23 @@ Chcete-li nám pomáhat na dalším rozvoji projektu, vyplňte formulář na [ce
 - Mac Catalyst (TODO)
 - Apple Watch (9+)
 
-# Hacking
+# Contributing
 
-> Jestli chceš přidat nové funkce, nebo máš nápady tak se připoj k nám na Slack.
+At first if you want to add new features or have ideas, join us on Slack.
 
-Základní použití je jednoduché:
+Basic usage is simple:
```

Comment:
`Then just clone the repository and open the project in XCODE and start hacking` ?